site/app/templates/grading/Images.twig

```diff
@@ -10,10 +10,12 @@
         {% if sections|length > 0 %}
             {% for registration, students in sections %}
                 <tr class="info">
-                    <td colspan="8" style="text-align: center">Students Enrolled in Registration Section {{ registration }}</td>
+                    <td colspan="5" style="text-align: center">Students Enrolled in Registration Section {{ registration }}</td>
                 </tr>
                 <tbody id="section-{{ registration }}">
+                {% set count = 0 %}
                 {% for student in students %}
+                    {% set count = 4 - (loop.index % 5) %}
                     {% if loop.index % 5 == 1 %}
                         <tr id="user-{{ student.getId() }}">
```

Comment:
Are we using this `id` anywhere or can it be moved to the `td` element which would be more appropriate?